master/internal/templates/api.go

```diff
@@ -48,6 +48,9 @@ func (a *TemplateAPIServer) GetTemplates(
 	if req.Name != "" {
 		q.Where("name ILIKE  ('%%' || ? || '%%')", req.Name)
 	}
+	if req.WorkspaceId != 0 {
```

Comment:
remind me, is 0 the uncategorized / default namespace, and should we allow filtering by it?

Reply:
Default workspace is 1